apps/passport-server/src/util/devconnectTicket.ts

```diff
@@ -29,7 +29,8 @@ export function pretixTicketsDifferent(
   }
 
   if (
-    oldTicket.pretix_checkin_timestamp !== newTicket.pretix_checkin_timestamp
+    oldTicket.pretix_checkin_timestamp?.getTime() !==
+    newTicket.pretix_checkin_timestamp?.getTime()
   ) {
```

Comment:
@robknight this was a bug I caught by observing in the logs that we were updating an awful lot of tickets, which didn't make sense to me. I recommend keeping an eye on the logs and rollbar.

Reply:
<img width="223" alt="Screenshot 2023-09-14 at 1 38 52 PM" src="https://github.com/proofcarryingdata/zupass/assets/2636237/63f261d1-c5df-4bfe-8534-686258319dfd">